shared/test/frontend_tester.py

```diff
@@ -17,14 +17,59 @@
 # need to consider whether this code wants to be shared with WebDriver tests
 # for other browsers, though.
 #
+
+gCommonDir = None
+
+# These redirects map the paths expected by the index.html files to the paths
+# in mozilla-central. In the github repo, the unit tests are run entirely within
+# karma, where the files are loaded directly. For mozilla-central we must map
+# the files to the appropriate place.
+REDIRECTIONS = {
+    "/test/vendor": "/chrome/content/shared/test/vendor",
+    "/shared/js": "/chrome/content/shared/js",
+    "/shared/test": "/chrome/content/shared/test",
+    "/shared/vendor": "/chrome/content/shared/vendor",
+    "/add-on/panels/vendor": "/chrome/content/panels/vendor",
+    "/add-on/panels/js": "/chrome/content/panels/js",
+    "/add-on/shared/js": "/chrome/content/shared/js",
+    "/add-on/shared/vendor": "/chrome/content/shared/vendor",
+}
+
+
 class ThreadingSimpleServer(SocketServer.ThreadingMixIn,
                             BaseHTTPServer.HTTPServer):
     pass
 
 
-class QuietHttpRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+class HttpRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
```

Comment:
A Python doc string for the class as a whole would be nice.  Doc strings for the individual methods probably aren't necessary.